### PR TITLE
Added TOC

### PR DESCRIPTION
**Pull request by OpenGlobe**
Added a bit of organization to make the guides a little easier to peruse, and added other relevant links from elsewhere in gov't.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,37 @@
 <p>18F Guides is the repository for best practices across our teams. 
 Our guides are open source, and you're free to use them as you wish.
 Our hope is that other digital service teams — both inside and outside the government — 
-will adopt or modify the practices outlined here. </p> 
+will adopt or modify the practices outlined here.</p>
+
+<p>Here's a curated list of guides found both here and elswhere that will help you get started:</p>
+
+##Management
```
Comment (mbland) [inline index.html:13]: Can you try to fashion these into the sidebar via the `navigation:` section in the `_config.yml` file per: https://pages.18f.gov/guides-template/add-a-new-page/make-a-child-page/

Also interested in what @emileighoutlaw, @kategarklavs, and/or @awfrancisco in particular think of this proposed architecture.

Comment (mbland) [inline index.html:13]: Whoops, just remembered that the Guides Template guides the reader to just use the `./go update_nav` script to produce the `navigation:` content. At any rate, you can view the `_config.yml` file in https://github.com/18F/guides-template/ as a model. Just note that you should have `internal: false` for any external links.

Comment (OpenGlobe) [inline index.html:13]: Yeah, I didn't know if it was a way you'd want to ultimately structure the guides or not, but it seems like grouping them somewhat might be helpful for the user.  If that is preferable I'd be happy to close this request and submit a new one w/ child pages, but Emileigh / Andrew might have a preferred way of organizing the content.

Also noticed that the guides aren't consistently named - some are "18F Guide for XX", others are just a doc name, might be nice to standardize?

Thanks!

Comment (awfrancisco) [inline index.html:13]: Definitely think we should have some name standardization, but that probably requires another conversation. 

Not sure if the open source guide belongs in Data. Maybe Dev? Also, if we have a Design grouping, then we should take Design out of "Dev and Design."

Comment (OpenGlobe) [inline index.html:13]: Makes sense to me - I made those changes to this pull request.

**Pull request closed without merging**